packages/docs/content/2_guidelines/3_design-principles.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: Design Principles
+parentNavItem: guidelines
```

Comment:
Is there a way to avoid having to include this `parentNavItem` information here?

Reply:
Ah! This was leftover from a previous implementation attempt. It looks like I can safely remove this now